app/queries/developer_query.rb

```diff
@@ -79,6 +78,17 @@ def query_and_paginate
     @pagy, @records = build_pagy(@_records, items: items_per_page)
   end
 
+  def build_pagy(collection, options = {})
```

Comment:
This needs a test in `test/queries/developer_query_test.rb` for this new behavior.

Reply:
Sure, I'm on it.